sp/metadata.php

```diff
@@ -39,13 +39,21 @@
     header('Content-Disposition: attachment; filename=' . $saml2auth->spname . '.xml');
 }
 
+if (is_siteadmin()) {
```

Comment:
does that mean we would need to download xml as admins and won't be able to provide it as URL?

Reply:
I removed this so it can be public